src/test/java/core/basesyntax/service/fileoperation/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.service.fileoperation.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.fileoperation.CsvFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.After;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+    private static final Path TEST_FILE_PATH = Path.of("src/test/resources.csv");
+    private static final Path INVALID_TEST_FILE_PATH = Path.of("src/test/invalidResources.csv");
+    private static final CsvFileReader reader = new CsvFileReaderImpl();
+    private static final List<String> validTestData = List.of(
+            "type,fruit,quantity",
+            "b,apple,100",
+            "b,banana,100",
+            "b,grapefruit,100",
+            "r,banana,1",
+            "s,apple,10",
+            "p,orange,10",
+            "s,grapefruit,10",
+            "r,orange,3");
+    private static final List<String> emptyTestData = List.of("type,fruit,quantity");
```

Comment:
```suggestion
    private static final CsvFileReader reader;
    private static final List<String> validTestData;
    private static final List<String> emptyTestData;
```
initialize in method with annotation @BeforeClass